argo-cd-apps/base/member/infra-deployments/backup/backup.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: backup
```

Comment:
sorry, I forgot one more thing. The development overlay will deploy both member and host ApplicationSet so since you added this one to the member, it will try to deploy it in dev, which we do not want, the ongoing test run should fail because of that.

You need to delete the backup ApplicationSet in `argo-cd-apps/overlays/development/delete-applications.yaml`

Reply:
the ApplicationSet is also in the production overlay, we need to either remove it from both dev and prod or find a way to add it only for staging